Solutions/Corvus.Identity.Examples.AzureFunctions/UsePlainTokensFunction.cs

```diff
@@ -19,26 +19,51 @@ namespace Corvus.Identity.Examples.AzureFunctions
     /// </summary>
     public class UsePlainTokensFunction
     {
-        private readonly ReadResourceGroupsWithPlainTokens client;
-        private readonly ILogger<ReadResourceGroupsWithPlainTokens> logger;
+        private readonly ReadResourceGroupsAsServiceIdentityWithPlainTokens serviceIdClient;
+        private readonly ILogger<ReadResourceGroupsAsServiceIdentityWithPlainTokens> logger;
         private readonly ExampleSettings settings;
+        private readonly ReadResourceGroupsWithIdentityFromConfigWithPlainTokens configIdClient;
 
         /// <summary>
         /// Creates a <see cref="UsePlainTokensFunction"/>.
         /// </summary>
         /// <param name="settings">Example settings.</param>
-        /// <param name="client">
-        /// The client wrapper for accessing the key vault.
+        /// <param name="serviceIdClient">
+        /// The client wrapper for reading from ARM using the service identity.
+        /// </param>
+        /// <param name="configIdClient">
+        /// The client wrapper for reading from ARM using a configured identity.
         /// </param>
         /// <param name="logger">Logger.</param>
         public UsePlainTokensFunction(
             ExampleSettings settings,
-            ReadResourceGroupsWithPlainTokens client,
-            ILogger<ReadResourceGroupsWithPlainTokens> logger)
+            ReadResourceGroupsAsServiceIdentityWithPlainTokens serviceIdClient,
+            ReadResourceGroupsWithIdentityFromConfigWithPlainTokens configIdClient,
+            ILogger<ReadResourceGroupsAsServiceIdentityWithPlainTokens> logger)
         {
-            this.client = client ?? throw new System.ArgumentNullException(nameof(client));
+            this.serviceIdClient = serviceIdClient ?? throw new System.ArgumentNullException(nameof(serviceIdClient));
```

Comment:
Any particular reason that `System.ArgumentNullException` is fully-qualified?

Reply:
No good reason. Will fix in https://github.com/corvus-dotnet/Corvus.Identity/issues/214